blog/_posts/aria-lanea-blog.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Planning for the Future of MABE2"
+date: 2021-08-06
+author: Aria Killibrew Bruehl and Lanea Rohan
+---
+
+## Outline
+
+# Planning for the future of MABE2: A summer of documentation and testing
+​
+Hello! This summer we (Lanea Rohan, '22 Grinnel College, and Aria Killebrew Bruehl,
+'23 Reed College) participated in the WAVES workshop through Michigan State University.
+We dedicated our summer to documenting and testing MABE2, an open-source research platform that provides accessible tools for conducting evolutionary computation and digital evolution research.
+​
+This project was guided by our wonderful mentors Acaica Ackles (documentation) and
+Austin Ferguson (testing).
+
+## Documentation
+​
+Prior to this summer there was no documentation for MABE2, meaning we had to tackle
+a huge library of files we had never interacted with before.
+​
+* documentation process (how we broke things down)
+..* obvious things to include (installation, quickstart)
+..* sales pitch (why use MABE2)
+..* additional things to help understanding (modules, orgs, etc)
+* what we could/couldn't document (because it wasn't done yet)
+* feedback process
+​
+You can view our documentation [here](https://mabe2.readthedocs.io/en/latest/).
+
+## Testing
+When you're going to be writing testing for a large project with multiple dependencies, the best place to start is by figuring out what you should test first. 
+
+- Levelization and bottom up
+    - Austin's vizualization of MABE2
+    - Assuming everything in Empirical was working correctly
+    - Flowchart process (insert the flowchart Austin made)
+        - Why use this?
+- Using CATCH2
+    - Words of Wisdom (what was easy/hard)
+        - Assert issue
```

Comment:
If you can quantify the amount of testing you accomplished (new lines of code covered, number of lines of tests written, etc. etc.) that would be great to include! Same comment goes for documentation